libguard/guard_interface.cpp

```diff
@@ -176,6 +176,16 @@ GuardRecord create(const EntityPath& entityPath, uint32_t eId, uint8_t eType)
                 existGuard.elogId = htobe32(eId);
                 file.write(offset + headerSize, &existGuard, sizeOfGuard);
             }
+            else if ((existGuard.errType == GARD_Predictive) &&
```

Comment:
same comment as previous commit

Reply:
> same comment as previous commit

Here, I am trying to convey "If we already have predictive guard record for the specific target and the later system attempts to create Fatal guard record for the same resource then, we said it already exists but, I think we should allow that since that might happen because of some error and **that's needed immediate action to fix** that by looking at the error but, that's not possible now since we are not allowing to override the predictive guard record by the Fatal.